data/VirtuozzoLinux.yaml

```diff
@@ -1,2 +1,2 @@
----
+lldpd::manage_repo: true
```

Comment:
can you please keep the `---` in the first line?

Reply:
ahh yes will add back